spark/dl/src/main/scala/com/intel/analytics/bigdl/python/api/PythonBigDL.scala

```diff
@@ -2889,6 +2893,12 @@ class PythonBigDL[T: ClassTag](implicit ev: TensorNumeric[T]) extends Serializab
     ImageFrameToSample[T](inputKeys.asScala.toArray, targets, sampleKey)
   }
 
+  def filesToImageFrame(url: String, sc: JavaSparkContext,
```

Comment:
suggest to change name to seqfilesToImageFrame, etc. Avoid to misunderstand to change image files to imageframe.